recipes/netcdf/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+from conans import CMake, ConanFile, tools
+import glob
+import os
+
+
+class NetcdfConan(ConanFile):
+    name = "netcdf"
+    description = "The Unidata network Common Data Form (netCDF) is an interface for scientific data access and a freely-distributed software library that provides an implementation of the interface."
+    topics = ("unidata", "unidata-netcdf", "networking")
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/Unidata/netcdf-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "netcdf4": [True, False],
+        "with_hdf5": [True, False],
+        "cdf5": [True, False],
+        "dap": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "netcdf4": True,
+        "with_hdf5": True,
+        "cdf5": True,
+        "dap": True,
+    }
+    generators = "cmake_find_package", "cmake_find_package_multi", "cmake"
+
+    _cmake = None
+
+    @property
+    def _with_hdf5(self):
+        return self.options.with_hdf5 or self.options.netcdf4
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if self._with_hdf5:
+            self.requires("hdf5/1.12.0")
+        if self.options.dap:
+            self.requires("libcurl/7.73.0")
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/20200517")
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("automake/1.16.2")
```

Comment:
If this is build by CMake are these required?

Reply:
No, remainder of my conan template